spec/features/right_word.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+describe 'test with wrong word' do
+
+  it 'right word' do
+    Card.create(original_text: "zzz", translated_text: "www", review_date: "01.01.1900")
+    visit root_path
+    fill_in 'user_answer', with: 'zzz'
+    click_button 'Проверить!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.